src/components/header.js

```diff
@@ -0,0 +1,19 @@
+import { AppBar } from "material-ui";
+import colors from "../styles";
+
+const Header = ({children, style}) => {
```

Comment:
see comment above... if you end up not needing children string, i think you have to remove children here because it will just be styles right?